hazelcast/src/test/java/com/hazelcast/internal/nearcache/impl/invalidation/ToHeapDataConverterTest.java

```diff
@@ -17,6 +17,7 @@
 package com.hazelcast.internal.nearcache.impl.invalidation;
 
 import com.hazelcast.internal.serialization.impl.HeapData;
+import com.hazelcast.internal.util.ToHeapDataConverter;
```

Comment:
The test should be moved as well.

Reply:
my ide needs some more intelligence. done.